src/plugins/intel_cpu/src/emitters/plugin/x64/jit_load_store_emitters.cpp

```diff
@@ -676,7 +676,7 @@ void jit_store_emitter::emit_data() const {
 }
 
 void jit_store_emitter::emit_impl(const std::vector<size_t> &in_idxs, const std::vector<size_t> &out_idxs) const {
-    const int offset = in_idxs.size() == 2 ? in_idxs[1] : 0;
+    const int offset = out_idxs.size() == 2 ? out_idxs[1] : 0;
```

Comment:
Could you please also add a description regarding offset for store and load emitter?

Reply:
Agree. Otherwise this is sacred knowledge which is known by a few developers